bower.json

```diff
@@ -12,5 +12,4 @@
   "license"      : "MIT",
   "main"         : "lib/jquery.raty.js",
   "name"         : "raty",
```

Comment:
You should to remove the last comma of the last line.
We have an invalid JSON here.